test/models/authenticatable_test.rb

```diff
@@ -30,12 +30,12 @@ class AuthenticatableTest < ActiveSupport::TestCase
 
   test 'find_or_initialize_with_errors adds blank error' do
     user_with_error = User.find_or_initialize_with_errors([:email], { email: "" })
-    assert_equal [:email, "can't be blank"], user_with_error.errors.first
+    assert_equal ["Email can't be blank"], user_with_error.errors.full_messages_for(:email)
```

Comment:
assert_equal ['Email can't be blank'], user_with_error.errors.full_messages_for(:email)

Use single quotations

Reply:
This string contains `'`, so we should escape it as back slash in the single quotations.
Therefore, using `"` makes it readable.

Why do you think so?

Reply:
Double quotes is fine here, thanks.